src/egithub_webhook.erl

```diff
@@ -12,7 +12,7 @@
 
 -export_type([request/0]).
 
--type request() :: #{headers => map(), body => map()}.
+-type request() :: #{headers => map(), body => binary()}.
```

Comment:
Not a fan of this. I do think we should do our best to ensure the decode function is called only once.